components/Converters/samples/BoolToVisibilityConverterSample.xaml

```diff
@@ -0,0 +1,27 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="ConvertersExperiment.Samples.BoolToVisibilityConverterSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:ConvertersExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <converters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter"
+                                              FalseValue="Collapsed"
+                                              TrueValue="Visible" />
```

Comment:
Are these not the default values?

Would probably be better to show the parameter being used to invert the effect (like have two images, one that's shown and one that's hidden) that way it shows the default usage as well as the inverted pattern with the `ConverterParameter`?